src/views/CustomToken.vue

```diff
@@ -257,14 +258,20 @@ export default {
     async addToken() {
       if (!this.existingAsset) {
         // Add only if it does not already exist
+        const gasLimit = getSendGasLimitERC20(this.chain)
+        if (!gasLimit) {
+          throw new Error(`${this.chain} doesn't support non native assets!`)
+        }
+
         await this.addCustomToken({
           network: this.activeNetwork,
           walletId: this.activeWalletId,
           chain: this.chain,
           contractAddress: this.contractAddress,
           name: this.name,
           symbol: this.symbol,
-          decimals: Number(this.decimals)
+          decimals: Number(this.decimals),
+          sendGasLimit: gasLimit
```

Comment:
The user has to add gasLimit property or it is fetched programatically. 
If use has to add it- how they know this property

Reply:
Gas limit is fetched from `crytoassets`. Method `getSendGasLimitERC20` returns gas limit based on chain id.